analyzers/tests/SonarAnalyzer.UnitTest/Rules/NormalizeStringsToUppercaseTest.cs

```diff
@@ -29,5 +29,16 @@ public class NormalizeStringsToUppercaseTest
         public void NormalizeStringsToUppercase() =>
             new VerifierBuilder<NormalizeStringsToUppercase>().AddPaths("NormalizeStringsToUppercase.cs")
                 .Verify();
+
+#if NET
+
+        [TestMethod]
+        public void NormalizeStringsToUppercase_CSharp11() =>
+            new VerifierBuilder<NormalizeStringsToUppercase>().AddPaths("NormalizeStringsToUppercase.CSharp11.cs")
```

Comment:
The guide line for the VerifierBuilder is to create a field
```cs
private readonly VerifierBuilder builder = new VerifierBuilder<DatabasePasswordsShouldBeSecure>();
```
and use it in the tests.
```suggestion
            builder.AddPaths("NormalizeStringsToUppercase.CSharp11.cs")
```